apps/admin_app/lib/admin_app_web/templates/product/index.html.eex

```diff
@@ -15,140 +14,133 @@
-              <button type="button" class="btn btn-outline-secondary">Renew</button>
-              <button type="button" class="btn btn-outline-secondary">Deactivate</button>
-              <button type="button" class="btn btn-outline-secondary">Delete</button>
-            </div>
-            <div class="btn-group mx-2">
-              <button class="btn btn-outline-secondary btn-md dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                Small button
-              </button>
-              <div class="dropdown-menu dropdown-menu-right">
-                <a class="dropdown-item" href="#">Action</a>
-                  <a class="dropdown-item" href="#">Another action</a>
-                  <a class="dropdown-item" href="#">Something else here</a>
-                  <div class="dropdown-divider"></div>
-                  <a class="dropdown-item" href="#">Separated link</a>
-              </div>
+        </div>
+        <div class=" col p-0">
+          <div class="btn-group" role="group" aria-label="Basic example">
+            <button type="button" class="btn btn-outline-secondary">Renew</button>
+            <button type="button" class="btn btn-outline-secondary">Deactivate</button>
+            <button type="button" class="btn btn-outline-secondary">Delete</button>
+          </div>
+          <div class="btn-group mx-2">
+            <button class="btn btn-outline-secondary btn-md dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Small button
+            </button>
+            <div class="dropdown-menu dropdown-menu-right">
+              <a class="dropdown-item" href="#">Action</a>
+              <a class="dropdown-item" href="#">Another action</a>
+              <a class="dropdown-item" href="#">Something else here</a>
+              <div class="dropdown-divider"></div>
+              <a class="dropdown-item" href="#">Separated link</a>
             </div>
           </div>
+        </div>
       </div>
       <div class="panel-group mt-3">
-      <div class="row">
-        <%= for item <- @products do %>
-        <div class="col-3 px-2 pb-3 m-0">
-        <div class="card p-0">
-          <img class="card-img-top" href="#<%= item.id %>" src="https://res.cloudinary.com/dnnqvqxz0/image/upload/v1532452255/empty-img.png" alt="Card image cap">
-          <div class="card-body">
-            <h5 class="card-title">  <%= item.name %></h5>
-            <div class="card-text">
-              <%= if has_variants(item) do %>
-                <div class="variants">
+        <div class="row">
+          <%= for item <- @products do %>
+          <div class="col-3 px-2 pb-3 m-0">
+            <div class="card p-0">
+              <img class="card-img-top" href="#<%= item.id %>" src="https://res.cloudinary.com/dnnqvqxz0/image/upload/v1532452255/empty-img.png" alt="Card image cap">
```

Comment:
@SagarKarwande this image should be picked from assets shouldn't it? since this the placeholder image, am I missing something

Reply:
No sir , i have miss-out this think also i have remove online links and replaced with local url's